### PR TITLE
feature/city-rating

### PR DESCRIPTION
**Pull request by hhrynenko**
feat: city rating, controller files rename, database models update and some routes files separate.

### CODE REVIEW

File: controllers/comment.controller.js [hunk 1]
```diff
@@ -63,12 +63,17 @@ const getByCity = async (req, res) => {
...
     try {
-        if (isEmpty(req.body)) {
+        if (isEmpty(req.body.cityName)) {
             return res.status(500).json({
-                error: 'Request body is empty.',
+                error: 'Request body is not full.',
+            });
+        }
+        const { cityName, commentText, commentGrade } = req.body;
+        if (commentGrade > 10 || commentGrade < 1 || Number.isInteger(commentGrade)) {
```
Comment (nikita-stovpak) [inline controllers/comment.controller.js]: ``` Number.isInteger(commentGrade) ```
you got number and your return the error ?

File: controllers/comment.controller.js [hunk 2]
```diff
@@ -79,10 +84,32 @@ const addComment = async (req, res) => {
                 error: `There are no ${cityName} in list. For full list see: /api/cities/`,
             });
         }
-        await Comment.create({
+        const ifCommentInDb = await Comment.create({
```
Comment (nikita-stovpak) [inline controllers/comment.controller.js]: if variable ex: 'ifCommentInDb' shoud be bool
if variable is object, array - it is commentInDb and them you check is variable as you wish

File: controllers/comment.controller.js [hunk 3]
```diff
@@ -110,14 +137,47 @@ const updateComment = async (req, res) => {
...
+            });
+            const allGradesByCity = await Comment.findAll({
+                attributes: ['cityGrade'],
+                where: {
+                    cityId: cityInDb.cityId,
+                },
+            });
+            const result = allGradesByCity
+                    .reduce((partialSum, grade) => partialSum + grade.dataValues.cityGrade, 0)
+                / allGradesByCity.length;
```
Comment (nikita-stovpak) [inline controllers/comment.controller.js]: fix spaces

File: middleware/validateRequest.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-const checkApiKey = (req, res, next) => {
+const validateRequest = (req, res, next) => {
```
Comment (nikita-stovpak) [inline middleware/validateRequest.js]: authMiddleware

**Review by stovpak — APPROVED** [review]

**Pull request merged**